src/main/java/net/glowstone/io/entity/FireballStore.java

```diff
@@ -0,0 +1,45 @@
+package net.glowstone.io.entity;
+
+import java.util.Arrays;
+import java.util.List;
+import net.glowstone.entity.projectile.GlowFireball;
+import net.glowstone.util.nbt.CompoundTag;
+import net.glowstone.util.nbt.TagType;
+import org.bukkit.util.Vector;
+
+public class FireballStore<T extends GlowFireball> extends ProjectileStore<T> {
+
+    public static final String IS_INCENDIARY = "X-Glowstone-IsIncendiary";
```

Comment:
Could you make these private like the other ones? Also, I feel like `glowstone:IsIncendiary` or if that's not valid, `glowstoneIsIncendiary` would be better.

Reply:
Made private and changed to "glowstone:IsIncendiary" and "glowstone:ExplosionPowerFloat" in aaadd0e2356f861f5a5090c2d63ee61f501c23f2.